notesapi/v1/tests/test_views.py

```diff
@@ -92,10 +92,10 @@ def _get_search_results(self, **kwargs):
         """
         Helper for search method. All keyword parameters are passed in GET
         """
-        q = QueryDict("user=" + TEST_USER, mutable=True)
-        q.update(kwargs)
-        url = reverse('api:v1:annotations_search') + '?{}'.format(q.urlencode())
-        result = self.client.get(url)
+        data = {"user": TEST_USER}
+        data.update(kwargs)
+        url = reverse('api:v1:annotations_search')
+        result = self.client.get(url, data=data)
```

Comment:
Removed `QueryDict ` due to UnicodeEncodeErrors. `data` param to get method will be treated as [query_string](https://github.com/tomchristie/django-rest-framework/blob/master/rest_framework/test.py#L82])